### PR TITLE
aded feature for quit input to accept both lowercase and uppercase q

### PR DESCRIPTION
**Pull request by chrismon88**
@Hockeydan25 Hey man got the input for the quit menu to allow both lower and uppercase please review.

### CODE REVIEW

**Comment by Hockeydan25** [comment]
@chrismon88  testing the code now. thanks for the update.

**Comment by Hockeydan25** [comment]
@chrismon88 if you have no other edits we can merge your issue.

**Comment by Hockeydan25** [comment]
@ merged issue fix, working well for the program. #13. Do a git pull on main for latest update/version.

**Review by claraj — COMMENTED** [review]
There's a number of extra changes that aren't related to the issue in this PR. Make sure you check out main before creating a new branch

File: main.py [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 from bookstore import Book, BookStore
 from menu import Menu
-from bookstore import BookError
```
Comment (chrismon88) [inline main.py:5]: @claraj add this back in?

File: main.py [hunk 2]
```diff
@@ -36,8 +36,8 @@ def add_book():
     try:
         new_book = ui.get_book_info()
         new_book.save()
-    except BookError:
```
Comment (chrismon88) [inline main.py:39]: @claraj add BookError back in to my case_sensitive branch?

File: main.py [hunk 3]
```diff
@@ -56,13 +56,13 @@ def show_all_books():
...
 def search_book():
-    search_term = ui.ask_question('Enter search term, will match partial authors or titles.')
+    search_term = ui.ask_question('Enter search term, will match partial authors or titles. ')
     matches = store.book_search(search_term)
     ui.show_books(matches)
 
 
 def change_read():
-
+    show_all_books()
```
Comment (chrismon88) [inline main.py:65]: @claraj remove show_all_books()method call?

**Pull request merged**